selfdrive/ui/tests/test_translations.py

```diff
@@ -121,6 +123,56 @@ def test_entities_error(self):
         matches = re.findall(r'@(\w+);', cur_translations)
         self.assertEqual(len(matches), 0, f"The string(s) {matches} were found with '@' instead of '&'")
 
+  def test_bad_language(self):
+    """
+    Tests if the given translation files contain any bad language using LDNOOBW.
+    """
+    AVAILABLE_LANGUAGE_CODES = ['ar', 'cs', 'da', 'de', 'en', 'eo', 'es', 'fa', 'fi', 'fil', 'fr', 'hi', 'hu', 'it', 'ja', 'kab', 'ko', 'nl', 'no', 'pl',
+                                'pt', 'ru', 'sv', 'th', 'tlh', 'tr', 'zh']
```

Comment:
we can remove this, let's just try to pull the file and let it raise an exception if it can't find it